pkg/log/logger.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2024 Outreach Corporation. All Rights Reserved.
+
+// Description: Implements a compatible API for the olog package and log package.
+
+package log
+
+import (
+	"context"
+	"io"
+	"log/slog"
+
+	"github.com/getoutreach/gobox/pkg/olog"
+	"github.com/sirupsen/logrus"
+)
+
+// Logger is the compatible log interface
+type Logger struct {
+	logrusLogger *logrus.Logger
+	ologLogger   *slog.Logger
+	useLogrus    bool
+}
+
+func New(useLogrus bool) *Logger {
```

Comment:
issue: this should probably be named something else, otherwise it gets confusing as to why you need to use this versus all of the other module functions.